### ISSUE TITLE
Add primitives from keras.layers and keras.models

### ISSUE THREAD

**Issue by itinawi** [HDI-Project/MLPrimitives#47]
### Description

I would like to add the following submodules to MLPrimitives:

From keras.layers
- keras.layers.LSTM (https://keras.io/layers/recurrent/#lstm)
- keras.layers.Dense (https://keras.io/layers/core/#dense)
- keras.layers.Dropout (https://keras.io/layers/core/#dropout)
- keras.layers.Input (https://keras.io/layers/core/#input)

From keras.models
- keras.models.Model (https://keras.io/models/model/)

**Comment by itinawi** [comment]
I will only be adding one file called

keras.Sequential.LSTMTimeSeriesRegressor.json

This file will include the Keras layers mentioned above and import the model from adapters/keras.py
